### PR TITLE
Update quantity on Signup Page

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
Updates the Signup Component so you can now update a post's quantity from there. 

#### How should this be reviewed?
👀 
Does this work as we expect? Does this update the post quantity and the overall signup quantity as we expect? 

#### Relevant tickets
Fixes [this](https://www.pivotaltracker.com/n/projects/2019429/stories/157094042) Pivotal Card.

#### How to deploy
https://github.com/DoSomething/rogue/blob/master/docs/development/deployments.md

### CODE REVIEW

File: resources/assets/components/Signup/index.js [hunk 1]
```diff
@@ -12,8 +12,7 @@ import Post from '../Post';
 import Empty from '../Empty';
 import Quantity from '../Quantity';
 import TextBlock from '../TextBlock';
-// @TODO: add this back in when we are ready to show history modal.
-// import HistoryModal from '../HistoryModal';
```
Comment (katiecrane) [inline resources/assets/components/Signup/index.js]: You can remove these commented lines!

Comment (chloealee) [reply resources/assets/components/Signup/index.js]: wait this is so weird @katiecrane ! It isn't showing up in my text editor so I'm not sure why it is here and all seems up to date...

Comment (chloealee) [reply resources/assets/components/Signup/index.js]: screen shot 2018-06-20 at 11 00 06 am

Comment (katiecrane) [reply resources/assets/components/Signup/index.js]: Are you sure you've pushed up the latest/don't have any uncommitted changes?

Comment (chloealee) [reply resources/assets/components/Signup/index.js]: oh wait - that is because they are removed! :)

Comment (katiecrane) [reply resources/assets/components/Signup/index.js]: omg WHAT IS WRONG WITH ME

Comment (chloealee) [reply resources/assets/components/Signup/index.js]: I was tricked too!! 😂

**Pull request merged**